client/src/main/java/org/apache/rocketmq/client/impl/consumer/RebalanceLitePullImpl.java

```diff
@@ -150,6 +151,11 @@ public long computePullFromWhereWithException(MessageQueue mq) throws MQClientEx
                 break;
             }
         }
+
+        if (result < 0) {
+            throw new MQClientException(ResponseCode.SYSTEM_ERROR, "Found unexpected result " + result);
+        }
+
```

Comment:
Many case will cause result equals to -1,  It should be very carefully to throw a exception before it is does a bug. 
It is not the same as method RebalancePushImpl#computePullFromWhereWithException